bootstrap.py

```diff
@@ -78,6 +78,9 @@ def main(args):
     shutil.copy(src, dst)
     return 0
 
+  if not shutil.which('npm'):
+    utils.exit_with_error('npm not found in path.')
```

Comment:
We also depend on `git` .. maybe check for that too.   Or maybe we make the check automatic/dynamic based on the actions?

lgtm either way though.

Reply:
Actually why don't remove the `shutil.which` commands a above, and then check `if not os.path.isabs(cmd[0]): .. find with which or fail`

Reply:
Good idea, done.